src/main/java/com/bookclub/iao/IEventAO.java

```diff
@@ -0,0 +1,72 @@
+package com.bookclub.iao;
+
+import com.bookclub.model.Event;
+
+import java.util.List;
+
+public interface IEventAO {
+    /**
+     * Retrieves a list of all events from the database
+     * @return All events found in the database, or null if not found.
+     */
+    public List<Event> findAllEvents() 
```

Comment:
Nice, build scripts doing its job.

```suggestion
    public List<Event> findAllEvents();
```